### PR TITLE
Switched around/added icons

### PR DESCRIPTION
**Pull request by mpage21**
GWAS studies to GWA studies

### CODE REVIEW

**Comment by mpage21** [comment]
Screenshot (209)

Here is a visual of the updated icons

File: static/index.html [hunk 1]
```diff
@@ -142,6 +127,20 @@ <h5>Cite the PRS Knowledge Base</h5>
...
             </div>
+            <div class="tile" onclick="location.href='cli_download.html';" style="cursor: pointer;">
+              <div>
+                <img class="tile-icon" src="/images/Icons/CLIicon.svg" alt="command line icon"
+                  onContextMenu="return false;">
+                <div class="middle"></div>
+              </div>
+              <div class="tile-content">
+                <h5>Download CLI tool</h5>
+                <!-- <p>Download our API to submit your VCF files to the analysis pipeline via the command line.</p> -->
```
Comment (MattCloward) [inline static/index.html:138]: Can this comment be removed?

**Review by MattCloward — CHANGES_REQUESTED** [review]
This looks great! See my one comment.

File: static/index.html (hunk 1 above)
Comment (mpage21) [reply static/index.html:138]: yes

**Pull request merged**